experiments/RLCD/AddSelfIdentity/to_hf.py

```diff
@@ -0,0 +1,26 @@
+import datasets
+import json
+
+
+if __name__ == '__main__':
```

Comment:
Do we have quantitative results for how well this worked over e.g. just SFT? Does RLCD help StableBeluga remember its creator?

Reply:
definitely worked to remember its creator, but no real quantitative results with this toy experiment. Tried just a normal SFT and it seemed to work too, but as this is such an easy example I'm not too surprised